.puppet-lint.rc

```diff
@@ -1,3 +1,3 @@
 --fail-on-warnings
---no-80chars-check
+--no-140chars-check
```

Comment:
This is managed through https://github.com/Katello/foreman-installer-modulesync which most like needs a similar change to https://github.com/theforeman/foreman-installer-modulesync/pull/36